distkeras/workers.py

```diff
@@ -262,13 +287,19 @@ def train(self, worker_id, iterator):
         self.pull()
         self.model.set_weights(self.center_variable)
         try:
+            sys.stderr.write("Debug: starting optimize...\n")
```

Comment:
Could this debugging information be removed?

Reply:
Just commented the debugging message